code/modules/food/food/snacks_vr.dm

```diff
@@ -678,3 +681,85 @@
 
 /obj/item/pizzabox/meat/Initialize()
 	pizza = new /obj/item/weapon/reagent_containers/food/snacks/sliceable/pizza/meatcargo(src)
+
+// food cubes
+/obj/item/weapon/reagent_containers/food/snacks/cube
+	name = "protein cube"
+	desc = "A colony of meat cells, Just add water!"
+	icon = 'icons/obj/food_vr.dmi'
+	icon_state = "proteincube"
+	w_class = ITEMSIZE_TINY
+	flags = OPENCONTAINER
+	bitesize = 12
+	filling_color = "#ADAC7F"
+	center_of_mass = list("x"=16, "y"=14)
+
+	var/food_type = "/obj/item/weapon/reagent_containers/food/snacks/proteinslab"
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/Initialize()
+	. = ..()
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/proc/Expand()
+	src.visible_message("<span class='notice'>\The [src] expands!</span>")
+	new food_type(get_turf(src))
+	qdel(src)
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/on_reagent_change()
+	if(reagents.has_reagent("water"))
+		Expand()
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/protein
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/protein/Initialize()
+	. = ..()
+	reagents.add_reagent("meatcolony", 5)
+
+/obj/item/weapon/reagent_containers/food/snacks/proteinslab
+	name = "Protein slab"
+	desc = "A slab of near pure protein, extremely artificial, and thoroughly disgusting."
+	icon = 'icons/obj/food_vr.dmi'
+	icon_state = "proteinslab"
+	bitesize = 10
+	nutriment_amt = 5
+	nutriment_desc = list("bitter chyme" = 50)
+
+/obj/item/weapon/reagent_containers/food/snacks/proteinslab/Initialize()
+	..()
+	reagents.add_reagent("protein", 30)
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/nutriment
+	name = "Nutriment cube"
+	desc = "A colony of plant cells, Just add water!"
+	icon_state = "nutrimentcube"
+	food_type = "/obj/item/weapon/reagent_containers/food/snacks/nutrimentslab"
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/nutriment/Initialize()
+	. = ..()
+	reagents.add_reagent("plantcolony", 5)
+
+/obj/item/weapon/reagent_containers/food/snacks/nutrimentslab
+	name = "Nutriment slab"
+	desc = "A slab of near pure plant-based nutrients, extremely artificial, and thoroughly disgusting."
+	icon = 'icons/obj/food_vr.dmi'
+	icon_state = "nutrimentslab"
+	bitesize = 10
+	nutriment_amt = 20
+	nutriment_desc = list("compost" = 50)
+
+/obj/item/weapon/reagent_containers/food/snacks/nutrimentslab/Initialize()
+	..()
```

Comment:
Shouldn't there be an add reagent here like the protein one?

Reply:
Nope, that's all in the nutriment_amt. It's just a pure block of 40 nutriment. It does however look like this line isn't needed.